### PR TITLE
data: thekrprince

### PR DESCRIPTION
**Pull request by thekrprince**

### CODE REVIEW

File: data/thekrprince.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/TheKrPrince",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/thekrprince/",
+      "icon": "FaLinkedIn"
```
Comment (kumarsonsoff3) [inline data/thekrprince.json:14]: You're probably be using an invalid icon here, It should be `FaLinkedin`.
```suggestion
      "icon": "FaLinkedin"
```

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by khushalarora — APPROVED** [review]
Looks good, thanks  :tada:

**Review by himanshu1221 — APPROVED** [review]
LGTM :+1:

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Welcome to LinkFree 🎉

**Pull request merged**